From 1723de89a4365a040eb517799807fe9bdaf9abf4 Mon Sep 17 00:00:00 2001
From: Ping Gao <ping.a.gao@intel.com>
Date: Mon, 12 Jun 2017 10:06:26 +0800
Subject: [PATCH 0461/1240] drm/i915/gvt: enlarge the scheduling period

Change the scheduling period from 1ms to 5ms, 1ms is too frequent
for APL as its CPU resource is limited.

Change-Id: I2b2cecdc3640db52a709cd33e39ca11c7b3292c4
Signed-off-by: Ping Gao <ping.a.gao@intel.com>
Acknowledged-by: Singh, Satyeshwar <satyeshwar.singh@intel.com>
Reviewed-by: He, Min <min.he@intel.com>
Reviewed-by: Jiang, Fei <fei.jiang@intel.com>
Reviewed-by: Dong, Eddie <eddie.dong@intel.com>
Tested-by: Dong, Eddie <eddie.dong@intel.com>
---
 drivers/gpu/drm/i915/gvt/sched_policy.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/gvt/sched_policy.c b/drivers/gpu/drm/i915/gvt/sched_policy.c
index 03532df..64356ad 100644
--- a/drivers/gpu/drm/i915/gvt/sched_policy.c
+++ b/drivers/gpu/drm/i915/gvt/sched_policy.c
@@ -194,7 +194,7 @@ static struct intel_vgpu *find_busy_vgpu(struct gvt_sched_data *sched_data)
 }
 
 /* in nanosecond */
-#define GVT_DEFAULT_TIME_SLICE 1000000
+#define GVT_DEFAULT_TIME_SLICE 5000000
 
 static void tbs_sched_func(struct gvt_sched_data *sched_data)
 {
-- 
2.7.4

